tests/testthat/test-querystring.R

```diff
@@ -7,7 +7,7 @@ test_that("query strings are properly parsed", {
 })
 
 test_that("special characters in query strings are handled properly", {
```

Comment:
Adding a test to make sure path parameters are not converted to spaces. (Just for reassurance)

```suggestion
test_that("path parameters do not convert + to space", {
  r <- plumber$new(test_path("files/path-params.R"))
  expect_equal(r$route(make_req("GET", "/car/a+b"), PlumberResponse$new()), "a+b")
})

test_that("special characters in query strings are handled properly", {
```

Reply:
Good, I did not think of that.